CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ information to effectively respond to your bug report or contribution.
 
 We welcome you to use the GitHub issue tracker to report bugs or suggest features.
 
-When filing an issue, please check [existing open](https://github.com/awslabs/aws-iot-device-sdk-c/issues), or [recently closed](https://github.com/awslabs/aws-iot-device-sdk-c/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), issues to make sure somebody else hasn't already 
+When filing an issue, please check [existing open](https://github.com/aws/aws-iot-device-sdk-embedded-C/issues), or [recently closed](https://github.com/aws/aws-iot-device-sdk-embedded-C/issues?q=is%3Aissue+is%3Aclosed), issues to make sure somebody else hasn't already 
```

Comment:
Remove commas and complete sentence
```suggestion
When filing an issue, please check [existing open](https://github.com/aws/aws-iot-device-sdk-embedded-C/issues) or [recently closed](https://github.com/aws/aws-iot-device-sdk-embedded-C/issues?q=is%3Aissue+is%3Aclosed) issues to make sure somebody else hasn't already opened a related issue.
```

Reply:
No, I think the commas are fine here. The sentence is continued on the next line:
```
When filing an issue, please check [existing open](https://github.com/aws/aws-iot-device-sdk-embedded-C/issues), or [recently closed](https://github.com/aws/aws-iot-device-sdk-embedded-C/issues?q=is%3Aissue+is%3Aclosed), issues to make sure someone else hasn't already 
reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
```

Reply:
![image](https://user-images.githubusercontent.com/5158611/93514589-3d1c9b80-f8dc-11ea-8478-ff74f3f64bac.png)

The "check existing open, or recently closed, issues" phrase seems incorrect. 

Shouldn't it be "check existing open, or recently closed  **_issues,_**"?

Reply:
No, I think since "recently closed" adds additional information, if you pause with either a `,` or `-`, then it should be both before and after the phrase. All of the following look fine to me:
```
Please check existing open - or recently closed - issues.
Please check existing open, or recently closed, issues.
Please check existing open (or recently closed) issues.
Please check existing open or recently closed issues.
```
I'd probably prefer either the first or last one, I just didn't think it's worth changing in this PR if the sentence already makes sense.